image_setup/02_devel_image/workspace_os_dependencies.txt

```diff
@@ -0,0 +1,4 @@
+# DO NOT EDIT (generated)
+#
+# This file is auto-generated by the write_osdeps.bash script
+# Install custom tools (e.g. editors) by adding them directly to the Dockerfile"

```

Comment:
renamed tools to packages. The tools in combination with editors did confuse me a little..